doc/whats_new/v0.24.rst

```diff
@@ -215,16 +215,23 @@ Changelog
 :mod:`sklearn.metrics`
 ......................
 
+- |Feature| Add :func:`metrics.cluster.pair_confusion_matrix` implementing
+  the confusion matrix arising from pairs of elements from two clusterings.
+  :pr:`17412` by :user:`Uwe F Mayer <ufmayer>`.
+
 - |Feature| Added :func:`metrics.mean_absolute_percentage_error` metric and
   the associated scorer for regression problems. :issue:`10708` fixed with the
   PR :pr:`15007` by :user:`Ashutosh Hathidara <ashutosh1919>`. The scorer and
   some practical test cases were taken from PR :pr:`10711` by
   :user:`Mohamed Ali Jamaoui <mohamed-ali>`.
 
-- |Fix| Fixed a bug in
-  :func:`metrics.classification_report` which was raising AttributeError
-  when called with `output_dict=True` for 0-length values.
-  :pr:`17777` by :user:`Shubhanshu Mishra <napsternxg>`
+- |Feature| :func:`metrics.plot_confusion_matrix` now supports making colorbar
+  optional in the matplotlib plot by setting colorbar=False. :pr:`17192` by
+  :user:`Avi Gupta <avigupta2612>`
```

Comment:
I'm wondering if something went wrong during a previous synchronisation with upstream... This block was below...

Reply:
@cmarmo The `what's new` instructions say: Entries should be grouped by module (in alphabetic order) and prefixed with one of the labels: |MajorFeature|, |Feature|, |Efficiency|, |Enhancement|, |Fix| or |API| ... Entries should be ordered by those labels (e.g. |Fix| after |Efficiency|).

I remember resorting an entry to follow those instructions during a manual merge conflict resolve. Currently I see:
```
- |Feature| Add :func:`metrics.cluster.pair_confusion_matrix` ...
- |Feature| Added :func:`metrics.mean_absolute_percentage_error` ...
- |Feature| :func:`metrics.plot_confusion_matrix` ...
- |Feature| Add :func:`metrics.rand_score`  ...
- |Enhancement| ...
```

Seem to be in the desired order, assuming the "Feature" label of Avi Gupta's metrics.plot_confusion_matrix entry is correct. Then again to me the making a color bar optional might be more of an "Enhancement" vs a "Feature". In that case the label should be changed from "Feature" to "Enhancement" and that block should be moved back down. What do you suggest?

Reply:
You are right: the issue here is that those modifications are not related with your pull request. If you decide to fix them it could be suitable to do that in a separate PR.

Reply:
Ok, I'll move it back down and I'll do a separate PR after this one is merged in (to avoid a PR on top of a PR).